src/server/src/main/java/io/cassandrareaper/service/RepairScheduleService.java

```diff
@@ -154,15 +154,17 @@ private void registerScheduleMetrics(UUID repairScheduleId) {
   }
 
   private void unregisterScheduleMetrics(UUID repairScheduleId) {
-    RepairSchedule schedule = context.storage.getRepairSchedule(repairScheduleId).get();
-    RepairUnit repairUnit = context.storage.getRepairUnit(schedule.getRepairUnitId());
```

Comment:
Wouldn't simple ``if(schedule != null)`` have sufficed? The optional's use for "isPresent + get" is considered a bad pattern. Not to mention there's multiple ``.get()`` calls. Optional works better (and more pretty) if used with a lambda:

```java
Optional<RepairSchedule> schedule = context.storage.getRepairSchedule(repairScheduleId);
schedule.ifPresent(s -> {
  RepairUnit repairUnit = context.storage.getRepairUnit(s).getRepairUnitId();
  // etc
});
```

Reply:
Sounds good, I'll use the lambda form 👍